src/base/projects/Header.svelte

```diff
@@ -195,18 +198,27 @@
       </div>
     {/if}
   </span>
-  <div class="stat commit-count clickable widget" class:active={content == ProjectContent.History} on:click={() => toggleContent(ProjectContent.History)}>
+  <div class="stat commit-count clickable widget" class:active={content == ProjectContent.History} on:click={() => toggleSource(ProjectContent.History)}>
     <strong>{tree.stats.commits}</strong> commit(s)
   </div>
   {#await Issue.getIssues(project.urn, seed.api) then issues}
-    <div class="stat issue-count clickable widget" class:active={content == ProjectContent.Issues} on:click={toggleIssues}>
+    <div class="stat issue-count clickable widget" class:active={content == ProjectContent.Issues} on:click={() => toggleContent(ProjectContent.Issues)}>
       <strong>{groupIssues(issues).open.length}</strong> issue(s)
     </div>
   {:catch}
     <div class="stat issue-count not-allowed widget" title="Not supported">
       0 issue(s)
     </div>
   {/await}
+  {#await Patch.getPatches(project.urn, seed.api) then patches}
+    <div class="stat patch-count clickable widget" class:active={content == ProjectContent.Patches} on:click={() => toggleContent(ProjectContent.Patches)}>
+      <strong>{groupPatches(patches).proposed.length}</strong> patche(s)
+    </div>
+  {:catch}
+    <div class="stat patch-count not-allowed widget" title="Not supported">
+      0 patche(s)
```

Comment:
IMO, since we won't render other values that aren't `commit | commits`, `patch | patches`, `contributor | contributors` or `issue | issues` in the project header, maybe we could show them without the parenthesis, wdyt?

We wouldn't need to create something complicated to define plurals for unknown words.